dom.bs

```diff
@@ -278,7 +278,8 @@ function imgFetched(ev) {
 <a>Event listeners</a> can be removed
 by utilizing the
 {{EventTarget/removeEventListener()}}
-method, passing the same arguments.
+method, passing the same arguments or by passing an {{AbortSignal}} to
+{{EventTarget/addEventListener()}}
```

Comment:
Suggested phrasing: break this into two sentences, i.e. the existing sentence as-is, plus another stating the alternative way.